pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -101,6 +101,10 @@ type NooBaaSpec struct {
 	// +optional
 	MongoDbURL string `json:"mongoDbURL,omitempty"`
 
+	// DebugLevel (optional) sets the debug level
+	// +optional
+	DebugLevel string `json:"debugLevel,omitempty"`
```

Comment:
why string and not a number?

Reply:
The fact that env variables are string only doesn't mean that CRD fields should make their schema fields to be all strings. Instead I would say that env vars should be converted to and from strings when needed.

Reply:
👍